### PR TITLE
Add gke staging upgrade test suites

### PR DESCRIPTION
**Pull request by krzyzacy**
I think we'll need to make a file pointer to `gs://kubernetes-release-gke/release/v1.8.0-gke.1`, I'll follow up on that.

/assign @foxish @ixdy

### CODE REVIEW

**Comment by foxish** [comment]
/lgtm

We need to change 1.8.0-gke.1 to 1.8.1-gke.0 at some point when we update staging.

**Comment by ixdy** [comment]
I'm confused by this; looking at the logs, the tests appear to be starting clusters at 1.7.6-gke.1, using tests from 1.8.0-gke.1, and then upgrading clusters to OSS `ci/latest` (i.e. 1.9-something), which is not what we want.

Am I misunderstanding?

**Comment by krzyzacy** [comment]
argh :-( yeah forgot to swap out the upgrade target

**Comment by foxish** [comment]
Slipped past me as well.. Thanks for verifying @ixdy

**Comment by krzyzacy** [comment]
(fix incoming)

File: jobs/config.json [hunk 1]
```diff
@@ -6891,6 +6891,69 @@
...
+      "--extract=gs://kubernetes-release-gke/release/v1.8.0-gke.1",
+      "--extract=gke",
+      "--gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging",
+      "--gcp-node-image=gci",
+      "--gcp-zone=us-central1-f",
+      "--gke-environment=staging",
+      "--provider=gke",
+      "--test_args=--ginkgo.skip=\\[Flaky\\]|\\[Feature:.+\\] --kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8",
+      "--timeout=900m",
+      "--upgrade_args=--ginkgo.focus=\\[Feature:ClusterUpgrade\\] --upgrade-target=ci/latest --upgrade-image=gci"
```
Comment (ixdy) [inline jobs/config.json]: should `--upgrade-target` be `v1.8.0-gke.1` instead?

**Pull request merged**